static/usage/alert/buttons/vue.md

```diff
@@ -42,7 +36,13 @@
 
         const { role } = await alert.onDidDismiss();
         this.roleMessage = `Dismissed with role: ${role}`;
-      },
+      };
+
+      return {
+        handlerMessage: '',
+        roleMessage: '',
```

Comment:
I get this error:

```
@ionic_vue.js?v=1be38721:4613 Uncaught TypeError: Cannot set properties of undefined (setting 'handlerMessage')
    at handler (Example.vue:28:22)
    at @ionic_vue.js?v=1be38721:4611:16
    at defaultGate (@ionic_vue.js?v=1be38721:4605:27)
    at safeCall (@ionic_vue.js?v=1be38721:4609:12)
    at proxyCustomElement.overlayIndex.callButtonHandler (@ionic_vue.js?v=1be38721:13599:26)
    at proxyCustomElement.overlayIndex.buttonClick (@ionic_vue.js?v=1be38721:13591:29)
    at HTMLButtonElement.onClick (@ionic_vue.js?v=1be38721:13684:188)
```

`handlerMessage` and `roleMessage` should be created using `ref('')` and assigned to variables. Then you can update them by doing `handlerMessage.value = 'foo'` instead of `this.handlerMessage = 'foo'`.